packages/frontend/src/components/MenuAlertDialog/index.tsx

```diff
@@ -25,27 +27,40 @@ interface AlertDialogContent {
   buttonText: string
 }
 
-function getAlertDialogContent(type: AlertDialogType): AlertDialogContent {
-  switch (type) {
+function getAlertDialogContent(
+  dialogHeader: string,
+  dialogType: AlertDialogType,
+): AlertDialogContent {
+  switch (dialogType) {
     case 'delete':
       return {
-        header: 'Delete Pipe',
-        body: "Are you sure you want to delete this pipe? You can't undo this action afterwards.",
+        header: `Delete ${dialogHeader}`,
+        body: `Are you sure you want to delete this ${dialogHeader?.toLowerCase()}? You can't undo this action afterwards.`,
         buttonText: 'Delete',
       }
     case 'duplicate':
       return {
-        header: 'Duplicate Pipe',
-        body: `You'll need to replace the data in every step and test each step in your duplicated pipe before publishing it.`,
+        header: `Duplicate ${dialogHeader}`,
+        body: `You'll need to replace the data in every step and test each step in your duplicated ${dialogHeader?.toLowerCase()} before publishing it.`,
```

Comment:
actually the description will be quite unique to duplicate pipe only right? maybe it should not need the dialog header variable here

Reply:
initially thought maybe parameterise in case there's a duplicate tile function in future, but agree we can change to var when we need next time